Source/Promise+Delay.swift

```diff
@@ -27,8 +27,9 @@ extension Promise {
 
 extension Promises {
     public static func delay(_ time: TimeInterval) -> Promise<Void> {
-        return Promise { (resolve: @escaping (() -> Void), _: @escaping ((Error) -> Void)) in
```

Comment:
This part complies fine without changing anything on my end, any reason why this changed?